index.html

```diff
@@ -0,0 +1,129 @@
+    <div class="tbl d-flex flex-md-row align-items-center gap-3">
+      <div class="row flex-lg-column row1"> 
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/book.svg" width="48" height="48" alt="book">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Reading Materials
+        </div>
+        <div class="col-6 col-md-12">
+          Offer references for reading materials to give context to your studies and to allow you to dig deep on your lessons.
+        </div>
+      </div>
+      <div class="row flex-md-column row2">
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/play.svg" width="48" height="48" alt="video">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Supporting Contents
+        </div>
+        <div class="col-6 col-md-12">
+          To aid your studies, we have compiled a list of tutorials which we deem best from the web.
+        </div>
+      </div>
+       <div class="row flex-md-column row3">
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/terminal.svg" width="48" height="48" alt="video">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Programming Tasks
+        </div>
+        <div class="col-6 col-md-12">
+          Coding tasks, mainly javascript function coding, along with the associative hints and answers.
+        </div>
+      </div>
+      <div class="row flex-md-column row3">
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/terminal.svg" width="48" height="48" alt="video">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Programming Tasks
+        </div>
+        <div class="col-6 col-md-12">
+          Coding tasks, mainly javascript function coding, along with the associative hints and answers.
+        </div>
+      </div>
+      <div class="row flex-md-column row3">
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/terminal.svg" width="48" height="48" alt="video">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Programming Tasks
+        </div>
+        <div class="col-6 col-md-12">
+          Coding tasks, mainly javascript function coding, along with the associative hints and answers.
+        </div>
+      </div>
+    </div>
+    <button type="button" class="sub m-3 mb-5 btn getstarted">Get started with the coding Challenges</button>
+  </section>
+  </div>
+  <script src="scripts/data.js"></script>
+  <script src="scripts/menu.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-A3rJD856KowSb7dwlZdYEkO39Gagi7vIsF0jrRAoQmDKKtQBHUuLZ9AsSv4jD4Xa" crossorigin="anonymous"></script>
+</body>
+</html>
```

Comment:
### Design Footer
![partner-footer](https://user-images.githubusercontent.com/93039359/181396266-6a63d9cf-7ec8-4758-9cca-d19f03e6f623.png)

**[Optional]**
- [ ] You didn't add the footer in your desktop version. Please add it :blush: to follow the design and  to make your design look professional :+1: